docs/HowToFAQ/FAQFunctionUnroll.md

```diff
@@ -0,0 +1,36 @@
+---
+title: A function seems to work just once. How do I get it to apply a bunch of times?
+---
+
+## Question
+
+A function seems to work just once. How do I get it to apply a bunch of times?
+Here is an example:
+```
+{% include_relative FAQFunctionUnroll0.dfy %}
+```
+
+The latter two lemmas will not prove without uncommenting the application of the lemma for one less iteration.
+How can I get this to prove automatically?
+
+## Answer
+
+Function bodies are transparent. That is, when there is a call to a function in Dafny code, Dafny will insert inline
+the definition of the function. But for recursive functions, it generally only does this once, as just once is 
```

Comment:
It's twice, not once; the function needs to be unfolded twice for sequences of length one, and three times for sequences of length 2, right?

Reply:
Not sure. But reworded to be less specific.

Reply:
I am sure: it is twice